_partials/_compression-intro.md

```diff
@@ -0,0 +1,23 @@
+When you enable compression, the data in your hypertable is compressed chunk by
```

Comment:
I'm really unsure about how this works as an intro. It reads more to me as "how compression works under the hood" than "what compression is and why you should use it".

Especially on the homepage, would a softer introduction to the topic work better? Is the first thing a reader wants to know about Timescale the shape of their compressed data? (It's very important later, just thinking about this in terms of the homepage)

Reply:
Yeah, I agree. I think we can improve this a lot.